src/Pair/toPairs.js

```diff
@@ -1,21 +1,20 @@
 /** @license ISC License (c) copyright 2017 original and current authors */
 /** @author Ian Hofmann-Hicks (evil) */
 
-const List = require('../core/List')
 const Pair = require('../core/Pair')
 const isObject = require('../core/isObject')
 
-// toPairs : Object -> List (Pair String a)
+// toPairs : Object -> Array (Pair String a)
 function toPairs(obj) {
   if(!isObject(obj)) {
     throw new TypeError('toPairs: Object required for argument')
   }
 
-  return Object.keys(obj).reduce(
-    (acc, key) => obj[key] !== undefined
-      ? acc.concat(List.of(Pair(key, obj[key])))
+  return Object.entries(obj).reduce(
```

Comment:
entries is supported by most, if not all, browsers and Node these days, but let me know if this should go back to keys

Reply:
Thinking we should go back to keys. JIC